doc/script_commands.txt

```diff
@@ -4375,13 +4375,15 @@ Leader:       All party members are warped to the leader's position. The
 If you specify a from_mapname, warpparty() will only affect those on
 that map.
 
+You can exclude Party leader from warping, by keeping include_leader value as 0.
```

Comment:
Here as well, can you change the `0` to `false`? I believe that using true/false as a boolean is more meaningful in this context, than using 1/0 as a number